openapi/schemas.yaml

```diff
@@ -259,6 +242,7 @@ components:
       title: limits-outgoing
       description: Open Payments specific property that defines the limits under which outgoing payments can be created.
       type: object
+      nullable: true
```

Comment:
I'm struggling to find `nullable` in OpenAPI or JSON Schema docs

Reply:
https://swagger.io/docs/specification/data-models/data-types/#null

Reply:
It looks like `nullable` was removed in OpenAPI 3.1
https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0